### PR TITLE
Increase index key prefixes size

### PR DESCRIPTION
**Pull request by SamuAlfageme**
See https://github.com/owncloud/core/blob/da5030c5c98ce563a4a4cd2b0e15c23e7ee71ed9/config/config.sample.php#L1121-L1151 and https://mariadb.com/kb/en/library/xtradbinnodb-server-system-variables/#innodb_large_prefix for the required MySQL setup to support long indexes (and therefore, 4byte characters)

Also, removing an unrelated duplicated in `my.cnf.tmpl`

### CODE REVIEW

**Comment by SamuAlfageme** [comment]
Btw, according to https://mariadb.com/kb/en/library/xtradbinnodb-server-system-variables/#innodb_large_prefix 

> `innodb_large_prefix`:
>  Default Value: ON (>= MariaDB 10.2.2) 

I guess this patch won't be needed as long as you rebuild the image with the `mariadb:latest` from https://hub.docker.com/_/mariadb/ since that's v10.2.9 already and `latest` in https://hub.docker.com/r/webhippie/mariadb/ is v10.1.28

```
docker exec server_db_1 mysql --version
mysql  Ver 15.1 Distrib 10.1.28-MariaDB, for Linux (x86_64) using readline 5.1
```

File: rootfs/etc/s6/mariadb/setup [hunk 1]
```diff
@@ -48,6 +52,9 @@ declare -x MARIADB_WRITE_BUFFER
 [[ -z "${MARIADB_INNODB_FLUSH_LOG_AT_TRX_COMMIT}" ]] && MARIADB_INNODB_FLUSH_LOG_AT_TRX_COMMIT="1"
 [[ -z "${MARIADB_INNODB_LOCK_WAIT_TIMEOUT}" ]] && MARIADB_INNODB_LOCK_WAIT_TIMEOUT="50"
 [[ -z "${MARIADB_INNODB_USE_NATIVE_AIO}" ]] && MARIADB_INNODB_USE_NATIVE_AIO="1"
+[[ -z "${MARIADB_INNODB_LARGE_PREFIX}" ]] && MARIADB_INNODB_LARGE_PREFIX="ON"
```
Comment (tboerger) [inline rootfs/etc/s6/mariadb/setup]: For the used version the default is `OFF`

File: rootfs/etc/s6/mariadb/setup [hunk 2]
```diff
@@ -48,6 +52,9 @@ declare -x MARIADB_WRITE_BUFFER
...
+[[ -z "${MARIADB_INNODB_FILE_FORMAT}" ]] && MARIADB_INNODB_FILE_FORMAT="Barracuda"
```
Comment (tboerger) [inline rootfs/etc/s6/mariadb/setup]: For the used version the default is `Antelope`

**Comment by tboerger** [comment]
> I guess this patch won't be needed as long as you rebuild the image with the mariadb:latest from https://hub.docker.com/_/mariadb/ since that's v10.2.9 already and latest in https://hub.docker.com/r/webhippie/mariadb/ is v10.1.28

This image have not been built from the scratch, it's based on alpine edge, and there we still got only 10.1.28.

**Pull request merged**